### PR TITLE
Feature/add host, exception, MDC[properties], and marker support

### PR DESCRIPTION
**Pull request by afinkenstadt**
Markers are now logged

### CODE REVIEW

**Comment by afinkenstadt** [comment]
Fixes #28

**Comment by afinkenstadt** [comment]
CLA signed.

**Comment by shakeelmohamed** [comment]
@afinkenstadt thanks for doing this work! We will take a look

File: README.md [hunk 1]
```diff
@@ -337,329 +337,3 @@ Introduction to the Splunk product and some of its capabilities
 ## Contact
 
 You can reach the Dev Platform team at [devinfo@splunk.com](mailto:devinfo@splunk.com).
```
Comment (glennblock) [inline README.md]: @afinkenstadt Can you refresh against master? The PR shouldn't show removing all of the content below if you are just making slight updates.

File: pom.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
   <groupId>com.splunk.logging</groupId>
   <artifactId>splunk-library-javalogging</artifactId>
-  <version>1.5.0</version>
+  <version>1.5.2-develop-3</version>
```
Comment (glennblock) [inline pom.xml]: Please revert the update to the version in pom.xml. Thanks!

File: README.md (hunk 1 above)
Comment (afinkenstadt) [inline README.md]: I'll figure that part out.  But this particular removal was due to duplicate sections in `README.md` in the first place.

Comment (afinkenstadt) [inline README.md]: image

**Comment by shakeelmohamed** [comment]
@afinkenstadt We've just merged your changed to `develop` in #37. We are working on getting these changes released, stay tuned!

**Comment by afinkenstadt** [comment]
Awesome!

File: src/main/java/com/splunk/logging/HttpEventCollectorEventInfo.java [hunk 3]
```diff
@@ -18,23 +18,46 @@
  * under the License.
  */
 
+import ch.qos.logback.classic.spi.IThrowableProxy;
```
Comment (shakeelmohamed) [inline src/main/java/com/splunk/logging/HttpEventCollectorEventInfo.java]: @afinkenstadt This isn't quite ready to go. Using `IThrowableProxy` adds a hard dependency on logback, which is something we definitely don't want (we also need support `java.util.logging` and log4) so forcing people to also depend on logback is not possible.

**Can we refactor this so references to `IThrowableProxy` are isolated to `HttpEventCollectorLogbackAppender.java`? If not, how do you feel about reverting those changes?**
This means removing `IThrowableProxy` from the following files:

* `HttpEventCollectorEventInfo.java`
* `HttpEventCollectorLog4jAppender.java` (unless there's a log4j equivalent)
* `HttpEventCollectorLoggingHandler.java`
* `HttpEventCollectorSender.java`

Comment (afinkenstadt) [inline src/main/java/com/splunk/logging/HttpEventCollectorEventInfo.java]: Perhaps #38 will address the inadvertent dependency taking on `ch.qos.logback`?

**Pull request merged**